WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipment Tracking Section/Add Tracking/ManualTrackingViewController.swift

```diff
@@ -547,6 +571,12 @@ private extension ManualTrackingViewController {
         ServiceLocator.stores.dispatch(action)
 
     }
+    func displayDismissConfirmationAlert() {
+        UIAlertController.presentDiscardChangesActionSheet(viewController: self,
+                                                           onDiscard: {[weak self] in self?.dismiss(animated: true)},
+                                                           onCancel: .none
```

Comment:
Do we need the `onCancel` parameter here? By default is nil, so I guess we can remove it altogether.